openday_scavenger/views/game/game.py

```diff
@@ -71,6 +71,14 @@ async def submit_answer(
 ):
     """AJAX style endpoint to submit the answer to a puzzle"""
 
+    # Check if visitor has already given a correct answer for this puzzle
```

Comment:
Other method - which prevents access to puzzle already correctly answered is probably better.